packages/expo-cli/src/commands/build/index.ts

```diff
@@ -125,8 +149,18 @@ export default function (program: Command) {
     )
     .asyncActionProjectDir(
       async (projectDir: string, options: AndroidOptions) => {
+        if (options.config) {
+          setCustomConfigPath(projectDir, options.config);
```

Comment:
This should not be necessary because the universal `--config` option is handled and `setCustomConfigPath` called in `asyncActionProjectDir`.

https://github.com/expo/expo-cli/blob/d1e02e31a4091ec43dd77053758f64c4b83ea00f/packages/expo-cli/src/exp.ts#L131-L137